app/grants/views.py

```diff
@@ -3928,3 +3737,217 @@ def upload_sybil_csv(request):
         return JsonResponse({'success': 'failed'}, status=500)
 
     return JsonResponse({'success': 'ok'}, status=200)
+
+
+class GrantSubmissionView(View):
```

Comment:
This is the new class based view that replaced the previous function based view. In the previous view there was a top level conditional checking if the request is a POST, all of the code that was nested in that clause is now in the `post` method below. The code that was in the `else` of the clause is in the `get` method below.